app/src/test/java/com/android/unio/model/event/EventViewModelTest.kt

```diff
@@ -73,4 +78,27 @@ class EventViewModelTest {
     viewModel.addEvent(
         inputStream, event1, { verify(repository).addEvent(eq(event1), any(), any()) }, {})
   }
+
+  @Test
+  fun testFindAssociationById() {
```

Comment:
I believe the name of this test has nothing to do with its contents, it might be better to rename it to something else.